client/rest/src/main/java/org/opensearch/client/RestClient.java

```diff
@@ -141,6 +143,30 @@ public class RestClient implements Closeable {
         NodeSelector nodeSelector,
         boolean strictDeprecationMode,
         boolean compressionEnabled
+    ) {
+        this(
+            client,
+            defaultHeaders,
+            nodes,
+            pathPrefix,
+            failureListener,
+            nodeSelector,
+            strictDeprecationMode,
+            compressionEnabled,
+            false // chunkedTransferEncodingEnabled
```

Comment:
I think this could alter the existing behavior: right now the `chunked` part is inherited from the request, but we may change it, wdyt @dblock ?

Reply:
Agreed. I made it an `Optional`. Maybe you have a better suggestion?